distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Base_Generator.enso

```diff
@@ -293,11 +293,13 @@ generate_from_part dialect from_spec = case from_spec of
         sub = generate_query dialect (Query.Select columns context)
         sub.paren ++ alias dialect as_name
 
-
 ## PRIVATE
 fold_case = lift_unary_op "FOLD_CASE" arg->
     code "LOWER(UPPER(" ++ arg ++ "))"
 
+## PRIVATE
+make_case_sensitive = lift_unary_op "MAKE_CASE_SENSITIVE" arg->arg
```

Comment:
Adding this fallback seems a little bit dangerous and I'm not certain if it is needed?

The default path will go through `Case_Sensitivity.Default` which will work correctly in Postgres.

If we don't have an implementation for 'force case sensitive' in Postgres, let's not pretend that we do have it.

I think the postgres `prepare_distinct` should simply raise `Unsupported_Database_Operation.Error` if asked to force case sensitivity, until the proper solution is implemented.